src/pcb/pcb_silkscreen_circle.ts

```diff
@@ -21,6 +22,7 @@ export const pcb_silkscreen_circle = z
     center: point,
     radius: length,
     layer: visible_layer,
+    stroke_width: distance,
```

Comment:
I think I missed this, use `length` and a default option of the thickness which is being used in the core right now